File: AssetManager/UserInterface/Forms/AdminTools/CrypterForm.vb
Public Class CrypterForm

    Sub New(ParentForm As Form)
        InitializeComponent()
        Tag = ParentForm
        Icon = ParentForm.Icon
        Show()
    End Sub

    Private Sub cmdEncode_Click(sender As Object, e As EventArgs) Handles cmdEncode.Click
        If Trim(txtString.Text) <> "" Then
            Dim CryptKey As String = Trim(txtKey.Text)
            Dim CryptString As String = Trim(txtString.Text)
            Using wrapper As New Simple3Des(CryptKey)
                txtResult.Text = wrapper.EncryptData(CryptString)
            End Using
        ElseIf Trim(txtResult.Text) <> "" Then
            Using wrapper As New Simple3Des(Trim(txtKey.Text))
                txtString.Text = wrapper.DecryptData(Trim(txtResult.Text))
            End Using
        End If
    End Sub

    Private Sub cmdClear_Click(sender As Object, e As EventArgs) Handles cmdClear.Click
        For Each ctl As Control In GroupBox1.Controls
            If TypeOf (ctl) Is TextBox Then
                Dim txt = DirectCast(ctl, TextBox)
                txt.Clear()
            End If
        Next
    End Sub

End Class

File: AssetManager/UserInterface/Forms/AdminTools/CrypterForm.cs
using AssetManager.UserInterface.CustomControls;
using System;
using System.Windows.Forms;

namespace AssetManager.UserInterface.Forms.AdminTools
{
    public partial class CrypterForm : ExtendedForm
    {
        public CrypterForm(ExtendedForm parentForm)
        {
            InitializeComponent();
            this.ParentForm = parentForm;
            Show();
        }

        private void cmdEncode_Click(object sender, EventArgs e)
        {
            if (!string.IsNullOrEmpty(txtString.Text.Trim()))
            {
                string CryptKey = txtKey.Text.Trim();
                string CryptString = txtString.Text.Trim();
                using (Simple3Des wrapper = new Simple3Des(CryptKey))
                {
                    txtResult.Text = wrapper.EncryptData(CryptString);
                }
            }
            else if (!string.IsNullOrEmpty(txtResult.Text.Trim()))
            {
                using (Simple3Des wrapper = new Simple3Des(txtKey.Text.Trim()))
                {
                    txtString.Text = wrapper.DecryptData(txtResult.Text.Trim());
                }
            }
        }

        private void cmdClear_Click(object sender, EventArgs e)
        {
            foreach (Control ctl in GroupBox1.Controls)
            {
                if ((ctl) is TextBox)
                {
                    var txt = (TextBox)ctl;
                    txt.Clear();
                }
            }
        }
    }
}